src/components/ListforOrganizing.css

```diff
@@ -0,0 +1,41 @@
+.SectionTipps {
+  display: flex;
+  flex-direction: column;
+}
+
+.TippsField {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  background-color: rgba(255, 255, 255, 0.8);
+  margin-top: 30px;
+  box-shadow: 0px 0px 5px 8px rgba(226, 221, 221, 0.3);
```

Comment:
reoccuring classes with the same rules like this boy shadow could be set as custom variables instead...